src/shogun/kernel/GaussianKernel.h

```diff
@@ -123,6 +125,15 @@ class CGaussianKernel: public CShiftInvariantKernel
 		return std::exp(m_log_width * 2.0) * 2.0;
 	}
 
+#ifndef SWIG
+	/**
+	 *	Returns a lazily evaluated Eigen expression template
```

Comment:
If it is already evaluated, then it won't be lazily evaluated (at least not from the point of view of deferring evaluation after this method has been executed).

What about without the lazily evaluated part, or refactoring to "to be lazily evaluated" if you really want to keep that part.

Reply:
yes, good point, i'll change that